### DISCUSSION TITLE
New to Hi.Events, interested in knowing more about a couple of features

### DISCUSSION THREAD

**Discussion by proffalken** [HiEventsDev/Hi.Events#233] [Q&A] [answered]
Hello,

I'm looking to migrate from alf.io to another platform and it's looking like it will be either hi.events or [Pretix](https://pretix.eu).

Both are AGPL licensed with similar conditions, but there are a few things I'm specifically looking for and I can't seem to see evidence of them in the documentation for Hi.Events:

1. SSO logins via OAuth2 - we use [member matters](https://github.com/membermatters/membermatters) to manage our membership at the hackspace including for centralised auth, and having the ability for users to see specific categories based on whether they are logged in or not would be amazing
2. Event attendance management - the docs say that [any device with a web browser](https://hi.events/docs/guides/managing-attendee-check-in#conducting-check-ins) can conduct a check-in via CheckIn lists (FWIW there is little information on how to do this such as how whether you need to be signed in etc. in the docs, I had to play around to work it out!).  In contrast, Pretix has [an entire hardware ecosystem](https://pretix.eu/about/en/features/checkin) or [dedicated apps](https://pretix.eu/about/en/scan) depending on preference to achieve this, as does alf.io. Are there plans to be able to integrate badge printers/barcode scanners into Hi.Events?
3.  Campaign Tracking - I may have missed this in the quick demo I've setup, but it would be great to be able to track *where* folks are coming from to purchase tickets so we know where to target in future.  This is something that [Pretix offers](https://pretix.eu/about/en/features/marketing), but Alf.io does not, so this is more of a "nice to have" than a requirement

So far this is really impressive, and way ahead of what Alf currently provides, so I'm keen to see if we can make this work for us, it just hinges on the above!

**Comment by islandmix** [comment]
I am not a developer but just my 2cents

#1... no opinion

#2.. I doubt a hardware ecosystem will be the priority... they are still in alpha phase... basic event ticketing functionality will probably be the priority. I think the baseline is functionality of eventbrite and everything else later... 

#3.. good idea to embed within tool.. but Facebook pixles and GoogleAnalytics can provide same results... so this also might be low on "day 1" functionality

**Comment by daveearley** [comment, marked as answer]
Hi @proffalken,

Thanks for trying out Hi.Events. To answer answer your questions:

SSO Login: We plan to implement basic SSO functionality soon, but category visibility based on login status isn't currently planned.

Event Check-in: We're developing a dedicated check-in app, but don't have immediate plans for hardware integration for check-in in the immediate future. We do however have plans in motion to integrate with badge printers.

Campaign Tracking: This feature is on our roadmap and should be available in the near future.

Thanks.

**Reply by proffalken** [reply]
Amazing, thanks @daveearley 

Is the roadmap available anywhere?

The links that I found on other answers seemed to point to a private repo and returned the GitHub 404.

**Reply by daveearley** [reply]
The roadmap should be public now. I had it set to private accidentally.
